README.md

```diff
@@ -2,34 +2,47 @@
 Pipeline for running single cell ENCODE E2G.
 
 Input: Multiome dataset (ATAC and RNA)
-Output: Enhancer -> Gene Predictions
+Output: Enhancer -> Gene Predictions (for each cell cluster)
 
 The pipeline consists of the following components:
-1. Compute ABC model predictions
+1. Compute ABC model predictions for each cell cluster
 2. Generate E2G features from ABC predictions
-3. Compute Kendall Correlation
-5. Combine 2 & 3 to construct a feature file to be used as input to the ML model
-6. Have ML model make predictions (assigning a score to each E-G pair)
+3. Compute Kendall correlation and/or ARC-E2G score for each cell cluster
+4. Combine 2 & 3 to construct a feature file to be used as input to predictive model
+5. (optional) Train predictive model using CRISPR-validated E-G pairs from K562 dataset
+6. Apply trained model to make predictions by assigning a score to each E-G pair
 
-## Usage
+## Set up
 Clone the repo and set it up for submdoule usage
 ```
 git clone --recurse-submodules https://github.com/EngreitzLab/sc-E2G.git
 git config --global submodule.recurse true
 ```
 
-Modify `config/config_biosamples.tsv` with your multiome data
-
 When running for the first time, the conda environments have to be setup.
 For speed, it's recommended that your current environment has mamba installed
 
 ```
 conda config --set channel_priority flexible  # Make sure your conda config uses flexible channel packaging to prevent unsatisfiable errors
-conda create -n mamba -c conda-forge mamba
+conda create -n mamba -c conda-forge mamba python=3.11
 conda activate mamba
-mamba install -c conda-forge -c bioconda snakemake
+mamba install -c conda-forge -c bioconda snakemake=7
 ```
 
+## Apply model
```

Comment:
Are the cell cluster configs also required for the model training workflow? Or not because that's just K562?